packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -901,6 +901,7 @@ export class Combobox
       this.emitComboboxChange();
       if (this.textInput) {
         this.textInput.value = item.textLabel;
+        console.log("toggle selection", item.textLabel);
```

Comment:
remove console log :)

Reply:
missed it here 65bd4651f0f459155a8c92bfe84d49971059b9d0